### PR TITLE
Move TCL test unit/type/bitmap to Go case

### LINKED ISSUE #823: Move TCL test unit/type/bitmap to Go case

### PR DESCRIPTION
**Pull request by tanruixiang**
it closes #823

### CODE REVIEW

**Comment by tanruixiang** [comment]
@git-hulk @PragmaTwice Hi. Do you have time to review for me? Thank you very much for your help.

**Comment by git-hulk** [comment]
> @git-hulk @PragmaTwice Hi. Do you have time to review for me? Thank you very much for your help.

Sure, thanks for your contribution.

**Review by git-hulk — DISMISSED** [review]

File: tests/gocase/unit/type/bitmap/bitmap_test.go [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+	for _, key := range keys {
+		cmd := rdb.Get(ctx, key)
+		require.NoError(t, cmd.Err())
+		buf = append(buf, cmd.Val())
+	}
+	return buf
+}
+func SimulateBitOp(op BITOP, values ...[]byte) string {
+	maxlen := 0
+	binarayArray := []string{}
```
Comment (PragmaTwice) [inline tests/gocase/unit/type/bitmap/bitmap_test.go]: `binary`?

Comment (tanruixiang) [reply tests/gocase/unit/type/bitmap/bitmap_test.go]: > `binary`?

It means 01 string. Name it according to `tcl` test.

Comment (PragmaTwice) [reply tests/gocase/unit/type/bitmap/bitmap_test.go]: I mean, it is a typo, binaray -> binary

Comment (tanruixiang) [reply tests/gocase/unit/type/bitmap/bitmap_test.go]: > I mean, it is a typo, binaray -> binary

After you pointed it out, I didn't even find this mistake. Thank you very much for your careful help. I've fixed it

**Review by PragmaTwice — APPROVED** [review]
LGTM

**Review by git-hulk — APPROVED** [review]

**Comment by tisonkun** [comment]
Thank you!

**Pull request merged**